Makefile

```diff
@@ -12,6 +12,8 @@ fast-cover:
 	go test -cover -tags="integration" $(TESTFLAGS)
 
 generate:
+	# Rename the package name in the proto definition to match the package
```

Comment:
This says what it does, but that was already fairly clear. It still isn't clear why this is necessary. I want it to say something like "this ensures that we don't forget to change the package name if we copy the proto definition from pilosa in order to update it. It is important that the packages have different names because proto can misbehave if pilosa and go-pilosa are imported into the same codebase."